src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/RemoteExecutorConsoleApp.cs

```diff
@@ -33,6 +33,16 @@ private static int Main(string[] args)
                 args.Subarray(3, args.Length - 3) : 
                 Array.Empty<string>();
 
+            // Unwrap any parameters wrapped in quotation marks.
+            for (int i = 0; i < additionalArgs.Length; i++)
+            {
+                string arg = additionalArgs[i];
+                if (arg.Length >= 2 && arg[0] == '"' && arg[arg.Length -1] == '"')
+                {
+                    additionalArgs[i] = arg.Substring(1, arg.Length - 2);
+                }
+            }
```

Comment:
Is this actually necessary?  I'm surprised they're still wrapped in quotes at this point... I'd have expected those quotes to be removed by the shell.  Is that not happening?